.github/workflows/causal_lm_cpp.yml

```diff
@@ -23,21 +24,20 @@ jobs:
       - name: Install OpenVINO
         run: |
           mkdir ./ov/
-          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.1/linux/l_openvino_toolkit_ubuntu20_2024.1.0.15008.f4afc983258_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
+          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/pre-release/2024.2.0rc1/linux/l_openvino_toolkit_ubuntu20_2024.2.0.dev20240524_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
           sudo ./ov/install_dependencies/install_openvino_dependencies.sh
       - name: Download, convert and build
         run: |
           source ./ov/setupvars.sh
-          python -m pip install --upgrade-strategy eager -r text_generation/causal_lm/cpp/requirements.txt
-          python -m pip install ./thirdparty/openvino_tokenizers/[transformers]
-          sudo apt-get install libtbb-dev
+          python -m pip install --upgrade-strategy eager -r ./samples/cpp/requirements.txt
```

Comment:
@Wovchena here we install older OV first, then override with newer one.
Should we change lines order? BTW, why not to install OV Tokenizers simple PyPi?

Reply:
They install identical OV. ./samples/cpp/requirements.txt has `--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release`